Sources/SwiftCrossUI/ViewGraph/PreferenceValues.swift

```diff
@@ -21,5 +52,13 @@ public struct PreferenceValues: Sendable {
                 }
             }
         }
+
+        // For presentation modifiers, take the first (top-level) value only
+        // This ensures only the root view's presentation modifiers apply to the sheet
+        presentationDetents = children.first?.presentationDetents
+        presentationCornerRadius = children.first?.presentationCornerRadius
+        presentationDragIndicatorVisibility = children.first?.presentationDragIndicatorVisibility
+        presentationBackground = children.first?.presentationBackground
+        interactiveDismissDisabled = children.first?.interactiveDismissDisabled
```

Comment:
Perform a merge over all of the children and take the first defined value when multiple children define a value for a given property. Similar to `navigationTitle`, I think these preferences should flow up the view hierarchy freely so that people can abstract out their presentation preferences into other views easily. E.g. adding spacers above and below the sheet's content should silently override the presentation modifiers set on the sandwiched content.

Something like `children.map(\.presentationDetents).first { $0 != nil }` should work well?